salt/_modules/metalk8s.py

```diff
@@ -82,7 +82,7 @@ def _execute_etcd_command(pod_name, cmd):
     api = kubernetes.client.CoreV1Api(api_client=client)
     etcd_command = [
         'etcdctl',
-        '--endpoints', 'https://localhost:2379',
+        '--endpoints', 'https://127.0.0.1:2379',
```

Comment:
About commit title: s/127.1/127.0.0.1/

Reply:
It is just a shorthand but ok, I will update